### PR TITLE
Fix Bug 983428: Add GA According To Spec

### PR DESCRIPTION
**Pull request by bensternthal**
Finished initial GA tagging.

Added Video share to GA

Removed uneeded ID

### CODE REVIEW

**Comment by bensternthal** [comment]
@sgarrity and others spec is here

https://docs.google.com/a/mozilla.com/presentation/d/1LAKpQFtw7oGHqgthbXlK2F86wN7hA8cRxzSy-iYnykw/edit?usp=sharing

**Comment by bensternthal** [comment]
Note, please merge when you can, rebasing this is icky. Also Gareth needs to test this on dev, he can't comment on the code.

File: mrburns/main/templates/number-modal.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <h1>{{ _('What does this number mean?') }}</h1>
         <p>
         {# Translators: Map Overlay #}
-        {% blocktrans with url='https://blog.mozilla.org/webdev/2014/04/29/the-web-we-want/' %}Anyone who uses Firefox helps shape the future of the Web and that's something we want to celebrate. That's why this number represents new downloads, automatic updates and any sharing that happens from this page &#150; the worldwide impact of our global community. If you're interested in how it all works, you can <a href="{{ url }}">check out the blog post.</a>{% endblocktrans %}
+        {% blocktrans with url='https://blog.mozilla.org/webdev/2014/04/29/the-web-we-want/' %}Anyone who uses Firefox helps shape the future of the Web and that's something we want to celebrate. That's why this number represents new downloads, automatic updates and any sharing that happens from this page &#150; the worldwide impact of our global community. If you're interested in how it all works, you can <a id="ga-modal-blog-post" href="{{ url }}">check out the blog post.</a>{% endblocktrans %}
```
Comment (pmclanahan) [inline mrburns/main/templates/number-modal.html]: This is going to break the string. Can you put an ID on the surrounding `<p>` in stead and select the child `<a>` w/ a selector?

Comment (pmclanahan) [inline mrburns/main/templates/number-modal.html]: alternately, you can get super hacky and do something like:

``` django
{% blocktrans with url='https://blog.mozilla.org/webdev/2014/04/29/the-web-we-want/" id="ga-modal-blog-post' %}
```

I prefer option 1, but 2 will work.

**Comment by bensternthal** [comment]
@pmclanahan ok fixed

**Comment by pmclanahan** [comment]
It's fine w/ me. Was hoping someone else would r+ the bulk of the JS, but we can land it for dev testing if you want.

**Comment by bensternthal** [comment]
I think we are OK to merge for testing, I would like to get Gareth testing.

**Comment by pmclanahan** [comment]
Fine with me. Rebase and merge at will.

**Pull request merged**